components/lib/tooltip/Tooltip.js

```diff
@@ -29,6 +29,8 @@ export const Tooltip = React.memo(
             }
         };
 
+        const [isBothFocus, setIsBothFocus] = React.useState(false);
```

Comment:
Move this line up to where the other `useState` values are so they are together. and call it `multipleFocusEvents`

Reply:
Okay, thanks for review!